capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/camera/photo/PhotoCropModifier.java

```diff
@@ -0,0 +1,91 @@
+package net.gini.android.capture.internal.camera.photo;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Matrix;
+import android.graphics.Rect;
+import android.util.DisplayMetrics;
+
+import java.io.ByteArrayOutputStream;
+
+public class PhotoCropModifier implements PhotoModifier {
+
+    private final Photo mPhoto;
+    private final int[] mScreenSize;
+    private final Rect aRect;
+
+    public PhotoCropModifier(Photo mPhoto, int[] mScreenSize, Rect aRect) {
+        this.mPhoto = mPhoto;
+        this.mScreenSize = mScreenSize;
+        this.aRect = aRect;
+    }
+
+    @Override
+    public void modify() {
+
+        if (mPhoto.getData() == null)
+            return;
+
+        synchronized (aRect) {
```

Comment:
[Request] Please use mPhoto as the monitor object. The other modifiers use it as well.